app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -783,15 +790,18 @@ private void closeTab(int index) {
           updateTabSwitcherIcon();
         })
         .show();
-    openDefaultTab();
-    updateTabSwitcherIcon();
+    if (webViewList.size() == 0) {
+      openDefaultTab();
+    }
+    else {
```

Comment:
`else` should be on the previous line. Follow the contributors' guidelines of this repo.

Reply:
Okay.
```suggestion
    else {
```

Reply:
Okay.